src/server/db/seed/index.ts

```diff
@@ -1,57 +1,64 @@
-const { PrismaClient } = require("@prisma/client");
-const { users, habits, checkIns, contacts } = require('./seedData.json');
+import { PrismaClient } from "@prisma/client";
+import { users, habits, routines} from './seedData.json';
 const prisma = new PrismaClient();
 
+interface HabitSeedData {
+    name: string
+    datesCompleted: string[]
+    checkInDay: string
+    ownerId: number
+}
+
+const habitsWithDateObjects = habits.map((habit: HabitSeedData) => {
+    return ({
+        ...habit,
+        datesCompleted: habit.datesCompleted.map((date: string) => new Date(date)) 
+    })
+})
+
+console.log("habitsWithDateObjects: ", habitsWithDateObjects);
+
 const load = async () => {
     try {
         await prisma.habit.deleteMany();
         console.log("Deleted records in habits table")
 
-        await prisma.checkIn.deleteMany();
-        console.log("Deleted records in checkIns table");
+        await prisma.routine.deleteMany();
+        console.log("Deleted records in routines table");
         
         await prisma.user.deleteMany();
         console.log("Deleted records in users table");
 
-        await prisma.contact.deleteMany();
-        console.log("Deleted records in contacts table");
-
         await prisma.$executeRaw`ALTER SEQUENCE "User_id_seq" RESTART WITH 1;`;
         console.log("reset user auto increment to 1")
 
         await prisma.$executeRaw`ALTER SEQUENCE "Habit_id_seq" RESTART WITH 1;`;
         console.log("reset habit auto increment to 1")
 
-        await prisma.$executeRaw`ALTER SEQUENCE "CheckIn_id_seq" RESTART WITH 1;`;
-        console.log("reset checkIn auto increment to 1")
-
-        await prisma.$executeRaw`ALTER SEQUENCE "Contact_id_seq" RESTART WITH 1;`;
-        console.log("reset contact auto increment to 1")
+        await prisma.$executeRaw`ALTER SEQUENCE "Routine_id_seq" RESTART WITH 1;`;
+        console.log("reset routine auto increment to 1")
 
         await prisma.user.createMany({
             data: users
         });
         console.log("Added user data");
 
         await prisma.habit.createMany({
-            data: habits
+            data: habitsWithDateObjects
         });
         console.log("Added habit data")
 
-        await prisma.checkIn.createMany({
-            data: checkIns
-        });
-        console.log("Added checkIn data")
-
-        await prisma.contact.createMany({
-            data: contacts
+        await prisma.routine.createMany({
+            data: routines
         });
-        console.log("Added contact data")
+        console.log("Added routine data")
 
     } catch(e) {
         console.error(e);
+        await prisma.$disconnect();
+        process.exit(1);
     } finally {
-        await prisma.$disconnect()
+        await prisma.$disconnect();
```

Comment:
It's my understanding that the finally block always runs, regardless of if the code hits the catch block or not. Not sure if it's a problem that it's awaiting disconnect after it would potentially hit that same command in the catch block.

Reply:
The catch block is there to log the error if there is one and it should be fine to then disconnect with the finally block. However, it would then execute `process.exit(0)` (which designates a success) even if there was an error. I wonder if we should change it to this:
```
    try {
        ...
        await prisma.$disconnect()
        process.exit(0);
    } catch(e) {
        console.error(e);
        process.exit(1);
    }

Reply:
This gave me the right exit code when I encountered an error. I think it's working well. Thanks for _catching_ this ;-)

Reply:
Great!